docs/en/stack/ml/df-analytics/ml-dfa-limitations.asciidoc

```diff
@@ -83,28 +125,17 @@ If any of these conditions applies, the destination index of the
 {dfanalytics-job} is deleted and the job starts again from the beginning – 
 regardless of the phase where the job was in.
 
-
-[float]
-[[dfa-training-docs]]
-== {dfanalytics-jobs-cap} cannot use more than 2^32^ documents for training
-
-A {dfanalytics-job} that would use more than 2^32^ documents for training cannot 
-be started. The limitation applies only for documents participating in training 
-the model. If your source index contains more than 2^32^ documents, set the 
-`training_percent` to a value that represents less than 2^32^ documents.
-
-
-[float]
+[discrete]
 [[dfa-missing-fields-limitations]]
-== Documents with missing values in analyzed fields are skipped
+=== Documents with missing values in analyzed fields are skipped
 
 If there are missing values in feature fields (fields that are subjects of the 
```

Comment:
I think we need to rephrase this a bit. Since it was added, the change is that we now support missing values depending on the analysis. So, `classification` and `regression` support missing values while `outlier_detection` does not.

We mention this just after, where we talk about limitations per analysis.

It might make more sense to remove this point and replace it with one that explains we skip rows where analyzed fields contain a multi-element array as a value. This limitation holds across analysis types.

Regarding missing values, it is probably best to only document it in the relevant section that is in the context of a specific analysis type (like we already do).